packages/docusaurus-theme-classic/src/theme/MDXComponents/index.tsx

```diff
@@ -31,6 +32,7 @@ const MDXComponents: MDXComponentsObject = {
   h4: (props) => <MDXHeading as="h4" {...props} />,
   h5: (props) => <MDXHeading as="h5" {...props} />,
   h6: (props) => <MDXHeading as="h6" {...props} />,
+  mermaid: MDXMermaid,
```

Comment:
Could you try with `@theme-init/` and move this to `theme-mermaid` instead? The classic theme shouldn't be dependent on `theme-mermaid`.

Reply:
Could you point me at an example for this please? I can't see from the documentation how to map the `mermaid` tag to the `MDXMermaid` component outside `MDXComponents`

Reply:
It would be the same idea as here: https://docusaurus.io/docs/next/markdown-features/react#mdx-component-scope

```ts
import MDXComponents from '@theme-init/MDXComponents';
import MDXMermaid from '@theme/MDXComponents/MDXMermaid';

export default {
  ...MDXComponents,
  mermaid: MDXMermaid,
};
```

Reply:
Thank you, this now works